src/compressed_tensors/config/format.py

```diff
@@ -108,33 +110,45 @@ def set_per_module_format(
                 "The provided format for the module does not match the "
                 "inferred format. Compression may fail "
             )
+    # If a per module format is not provided, check if a global format was provided
+    elif quantization_format is not None:
```

Comment:
could there be a case where both `module.quantization_scheme.format` is set and the `quantization_format` is also passed in? then this will result in a no-op?

Reply:
We use the quantization format that is passed in then. Essentially a global override

Reply:
Updated so that if a globbal format is provided, will override any per module formats